tests/v2/test_jsonrpc_waku.nim

```diff
@@ -102,7 +102,7 @@ procSuite "Waku v2 JSON-RPC API":
       response == true
     
     # Publish a message on the default topic
-    response = await client.post_waku_v2_relay_v1_message(defaultTopic, WakuRelayMessage(payload: @[byte 1], contentTopic: some(defaultContentTopic), timestamp: some(epochTime())))
+    response = await client.post_waku_v2_relay_v1_message(defaultTopic, WakuRelayMessage(payload: @[byte 1], contentTopic: some(defaultContentTopic), timestamp: some(int64(epochTime()))))
```

Comment:
I know for tests we don't care as much about the accuracy of the timestamps, but `int64(epochTime())` will convert the epochTime to a timestamp with _second_ resolution by simply stripping off the fractional part.
In order to avoid having to convert by `*1000` (or `*1000000000` if we move to ns), I think it's probably best to add a utility function somewhere that takes a `float64` timestamp (seconds resolution, with sub-seconds after the decimal) and convert it to `int64` in the desired resolution. In fact, if such a util doesn't already exist in a submodule, we can provide a `getNanosecondTime(): int64` function that even reads the epoch time on behalf of the module calling the function. This way the logic resides in one place and can be used by all tests and modules.